src/Screens/Main/BuddyList/index.tsx

```diff
@@ -46,6 +46,7 @@ export default ({ navigation }: Props) => {
     <TitledContainer
       TitleComponent={
         <RN.View style={styles.header}>
+          <RN.View style={{ height: 40, width: 40 }} />
```

Comment:
Avoid inline styles. They may feel handy but will slowly kill you from inside. It is a good habit to learn to avoid those.

Reply:
Please make some style for this style. It also names it nicely. It's kinda unclear now what this empty view is about

Reply:
If it is just a spacer, then you can call it spacer or something : ). Personally I am not a fan of empty spacer components to make layouts work but this is totally fine!